src/v3/src/hooks/useValue.ts

```diff
@@ -14,9 +14,9 @@ import { useWidgetContext } from '../contexts';
 import { FieldElement } from '../types';
 
 export const useValue = (uischema: FieldElement): unknown => {
-  const { name } = uischema.options.inputMeta;
+  const { name, value } = uischema.options.inputMeta;
 
   const { data } = useWidgetContext();
 
-  return data[name];
+  return data[name] ?? value;
```

Comment:
Should we limit this to only the identifier field or is the behavior in Gen2 to take values for any field from the server already as well?

Reply:
> is the behavior in Gen2 to take values for any field from the server already as well?

Yes, this behavior is implemented in this line: 
https://github.com/okta/okta-signin-widget/blob/2d7c17ab7044cb7bf94d5d0176adb80f3c5ba9c5/src/v2/view-builder/internals/BaseModel.js#L18
It picks value from IDX response to model. If I remove `'value'` from the list, Gen2 will not take values from the server by default